app/controllers/groups.js

```diff
@@ -113,15 +113,17 @@ module.exports = function(app) {
     Transaction
       .find({
         attributes: [
-          [sequelize.fn('SUM', sequelize.col('amount')), 'total']
+          [sequelize.fn('SUM', sequelize.col('netAmountInGroupCurrency')), 'total']
```

Comment:
Shouldn't the default be `inGroupCurrency`? In which case we shouldn't specify it?